src/index.css

```
@@ -233,3 +233,57 @@
   color: rgb(var(--c-fg));
   letter-spacing: 0.02em;
 }
+
+navbar {
```

Stylelint (17.12.0)

[error] Unknown type selector "navbar" (selector-type-no-unknown)

(selector-type-no-unknown)